securitycenter/docs/snippets_findings.py

```diff
@@ -0,0 +1,603 @@
+    # created previously (you can use list_sources to find a specific one).
+    # Its format is:
+    # source_name = "organizations/{organization_id}/sources/{source_id}"
+    # e.g.:
+    # source_name = "organizations/111122222444/sources/1234"
+    source = client.get_source(source_name)
+
+    print("Source: {}".format(source))
+    # [END get_source]
+
+
+def test_update_source(source_name):
+    """Updates a source's display name."""
+    # [START update_source]
+    from google.cloud import securitycenter as securitycenter
+    from google.protobuf import field_mask_pb2
+
+    client = securitycenter.SecurityCenterClient()
+
+    # Field mask to only update the display name.
+    field_mask = field_mask_pb2.FieldMask(paths=["display_name"])
+
+    # source_name is the resource path for a source that has been
+    # created previously (you can use list_sources to find a specific one).
+    # Its format is:
+    # source_name = "organizations/{organization_id}/sources/{source_id}"
+    # e.g.:
+    # source_name = "organizations/111122222444/sources/1234"
+    updated = client.update_source(
+        {"name": source_name, "display_name": "Updated Display Name"},
+        update_mask=field_mask,
+    )
+    print("Updated Source: {}".format(updated))
+    # [END update_source]
+    assert updated.display_name == "Updated Display Name"
+
+
+def test_add_user_to_source(source_name):
+    """Gives a user findingsEditor permission to the source."""
+    user_email = "csccclienttest@gmail.com"
+    # [START update_source_iam]
+    from google.cloud import securitycenter as securitycenter
+    from google.iam.v1 import policy_pb2
+
+    client = securitycenter.SecurityCenterClient()
+
+    # source_name is the resource path for a source that has been
+    # created previously (you can use list_sources to find a specific one).
+    # Its format is:
+    # source_name = "organizations/{organization_id}/sources/{source_id}"
+    # e.g.:
+    # source_name = "organizations/111122222444/sources/1234"
+    # Get the old policy so we can do an incremental update.
+    old_policy = client.get_iam_policy(source_name)
+    print("Old Policy: {}".format(old_policy))
+
+    # Setup a new IAM binding.
+    binding = policy_pb2.Binding()
+    binding.role = "roles/securitycenter.findingsEditor"
+    # user_email is an e-mail address known to Cloud IAM (e.g. a gmail address).
+    # user_mail = user@somedomain.com
+    binding.members.append("user:{}".format(user_email))
+
+    # Setting the e-tag avoids over-write existing policy
+    updated = client.set_iam_policy(
+        source_name, {"etag": old_policy.etag, "bindings": [binding]}
+    )
+
+    print("Updated Policy: {}".format(updated))
+
+    # [END update_source_iam]
+    assert any(
+        member == "user:csccclienttest@gmail.com"
+        for member in chain.from_iterable(
+            binding.members for binding in updated.bindings
+        )
+    )
+
+
+def test_list_source(organization_id):
```

Comment:
What if nobody has set up this organization before running tests?

Reply:
Then the tests fail.  I have a sample org that I need to enable access to python test service accounts for this repo.  Could you provide me with these and i'll can enable them?

Reply:
@engelke `snippets/noxfile.py` skips running the snippets if the `GCLOUD_ORGANIZATION` environment variable is not test.

@emkornfield The default CI account for the `google-cloud-python` repository is `precise-truck-742`.  Once that project has access to that organization, please have @busunkim96 or @crwilcox change the Kokoro setup to add that environment variable.

Reply:
@tseaver I need to full service account associated with the project to grant access, i.e. something like foo@precise-truck-742.iam.gserviceaccount.com

Reply:
@busunkim96 can you provide the right service account to @emkornfield here?

Reply:
@emkornfield I sent you an email with the full account name

Reply:
@crwilcox @busunkim96 I granted access to the account can you set the environment variable?

Reply:
ORG (1081635000895)

Reply:
I should set `GCLOUD_ORGANIZATION` to `1081635000895`?